source/_includes/site/sidebar.html

```diff
@@ -48,11 +48,37 @@ <h1 class="title delta">On this page</h1>
     {%- endunless -%}
   {% endif %} 
 
+  {%- if page.ha_platforms.first -%}
+  <section class="aside-module grid__item one-whole lap-one-half">
+    <div class='section'>
+      <h1 class="title delta">Platforms</h1>
```

Comment:
```suggestion
      <h1 class="title delta">Related building blocks</h1>
```
Replaced 'platform' by 'building block' to use consistent terminology. We are trying to avoid the term platform in the documentation.